python/ray/experimental/collective/allreduce.py

```diff
@@ -79,11 +78,25 @@ def __call__(
         self,
         tensor,
         group_name: str = "default",
-        op: ray_collective_types.ReduceOp = ray_collective_types.ReduceOp.SUM,
+        op: ReduceOp = ReduceOp.SUM,
     ):
         from ray.util.collective.collective import allreduce
+        from ray.util.collective.types import ReduceOp as RayReduceOp
+
+        def get_ray_reduce_op(op: ReduceOp) -> RayReduceOp:
```

Comment:
Please add a TODO here explaining why we do this.

Reply:
TODO added.